app/models/archived_patient.rb

```diff
@@ -71,19 +71,14 @@ class ArchivedPatient
 
   mongoid_userstamp user_model: 'User'
 
-  # Archive & Delete patients who have fallen out of contact
-  def self.archive_dropped_off_patients!
-    Patient.where(:initial_call_date.lte => 1.year.ago).each do |patient|
-      ArchivedPatient.convert_patient(patient)
-      patient.destroy!
-    end
-  end
-
-  # Archive & Delete patients who are completely through our process
-  def self.archive_fulfilled_patients!
-    Patient.fulfilled_on_or_before(120.days.ago).each do |patient|
-      ArchivedPatient.convert_patient(patient)
-      patient.destroy!
+  # Archive & delete audited patients who called a year or more ago, or any
```

Comment:
I have a question about this as related to patients who come back to dcaf for assistance a second time -- part of the purpose of archiving is handling this tough case. we don't have to hash this out here but before we merge let's talk thru the two year standard.